public/index.js

```diff
@@ -49,29 +49,33 @@ async function UpdateBusPositionMarkers()
         }
 
         markers = [];
+        selectedBtns = document.getElementsByClassName(CSS_CLASS_SELECTED);

```

Comment:
If this is a new variable can we use 'let' to define it? 'let selectedBtns = ...'